src/transformers/scss.ts

```diff
@@ -59,24 +38,17 @@ const transformer: Transformer<Options.Sass> = async ({
   filename,
   options = {},
 }) => {
-  let implementation = options?.implementation ?? sass;
```

Comment:
Since `node-sass` is deprecated, I think we might as well remove support for it altogether.